frontend/src/metabase/reference/databases/DatabaseList.jsx

```diff
@@ -40,6 +40,16 @@ export default class DatabaseList extends Component {
   render() {
     const { entities, style, loadingError, loading } = this.props;
 
+    const databases = Object.values(entities)
+      .filter(database => {
+        const exists = Boolean(database?.id && database?.name);
+        return exists && !database.is_saved_questions;
+      })
+      .sort((a, b) => {
+        const compared = a.name.localeCompare(b.name);
+        return compared !== 0 ? compared : a.engine.localeCompare(b.engine);
+      });
+
```

Comment:
Should we maybe move this entire logic to a selector? Can be a good opportunity to rename `entities` to `databases` here too

Reply:
Good idea; I'll do in a separate PR (clicked the merge button before I saw this comment 🙂 )